src/main/java/com/wolApi/wolApi/api/bibleVerses/BibleVerseRepository.java

```diff
@@ -0,0 +1,13 @@
+package com.wolApi.wolApi.api.bibleVerses;
+
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+import java.util.Optional;
+
+public interface BibleVerseRepository extends JpaRepository<BibleVerse, Long> {
+    @Query("SELECT b FROM BibleVerse b WHERE b.bookName = ?1 AND b.numChaptersInBook = ?2 AND b.numVersesInChapter = ?3")
+    BibleVerse findBibleVerseByBookNameAndChapterNumAndVerseNum(String bookName,
```

Comment:
Name too long. We have polymorphism so we can override the method. e.g. `findBibleVerse` can find by different things. The description can be found in the parameter names. I think naming to `findBibleVerse` is just fine.

Reply:
Funnily enough copilot did the naming for me 😆 

I think ill change it i just wanted to be clear about what is going on